integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/http/inbound/transport/test/HttpInboundDispatchTestCase.java

```diff
@@ -37,175 +30,64 @@
  */
 
 public class HttpInboundDispatchTestCase extends ESBIntegrationTest {
-    private LogViewerClient logViewerClient = null;
+
+    private static final String requestPayload = "<soapenv:Envelope xmlns:soapenv='http://schemas.xmlsoap.org/soap/envelope/' >"
+            + "<soapenv:Body xmlns:ser='http://services.samples' xmlns:xsd='http://services.samples/xsd'> "
+            + "<ser:getQuote> <ser:request> <xsd:symbol>WSO2</xsd:symbol> </ser:request> </ser:getQuote> "
+            + "</soapenv:Body></soapenv:Envelope> ";
+
+    private CarbonLogReader carbonLogReader;
 
     @BeforeClass(alwaysRun = true)
     public void setEnvironment() throws Exception {
         super.init();
-
-        logViewerClient = new LogViewerClient(contextUrls.getBackEndUrl(), getSessionCookie());
-
-        addSequence(getArtifactConfig("sequences", "fault.xml"));
-        addSequence(getArtifactConfig("sequences", "main.xml"));
-        addSequence(getArtifactConfig("sequences", "super.xml"));
-
-        addApi(getArtifactConfig("api", "BOO.xml"));
-        addApi(getArtifactConfig("api", "FOO.xml"));
-        addApi(getArtifactConfig("api", "TenantContext.xml"));
-
-        addProxyService(getArtifactConfig("proxy-services", "TestProxy.xml"));
-        addInboundEndpoint(getArtifactConfig("inbound-endpoints", "inbound1.xml"));
-        addInboundEndpoint(getArtifactConfig("inbound-endpoints", "inbound2.xml"));
+        carbonLogReader = new CarbonLogReader();
+        carbonLogReader.start();
 
     }
 
     @Test(groups = "wso2.esb", description = "Inbound HTTP Super Tenant Sequence Dispatch")
     public void inboundHttpSuperSequenceTest() throws Exception {
-        axis2Client.sendSimpleStockQuoteRequest("http://localhost:9090/", null, "WSO2");
+
+        HttpRequestUtil.doPost(new URL("http://localhost:9090/"), requestPayload, new HashMap<String, String>());
```

Comment:
Shall we get the hostname and the port from a config file?

Reply:
We have a method in ESBIntegrationTest ( super class ) to get the host name.

Reply:
Only hostname can be inferred as the post is defined in the [inbound synapse configuration](https://github.com/wso2/micro-integrator/blob/integration-tests-mediation-profile/integration/mediation-tests/tests-transport/src/test/resources/artifacts/ESB/http.inbound.transport/inbound1.xml#L10).

Fixed in https://github.com/wso2/micro-integrator/pull/580